tests/core/test_api/test_wrappers.py

```diff
@@ -1,19 +1,49 @@
 import requests
 
 from ai_transform.api.wrappers import request_wrapper
+from contextlib import redirect_stdout, redirect_stderr
+import io
+import sys
 
 
 class TestWrappers:
-    def test_request_wrapper_fail(self):
-        try:
+    def test_request_wrapper_fail_with_error_in_logs(self):
+        f = io.StringIO()
+        saved_stdout = sys.stdout
+        sys.stdout = f
+        resp = request_wrapper(
+            requests.post,
+            args=["https://www.google.com"],
+            num_retries=1,
+            timeout=1,
+            output_to_stdout=True,
+        )
+        output = f.getvalue()
+        sys.stdout = saved_stdout
+        assert "status_code" in output
+        assert "message" in output
+
+    def test_request_wrapper_fail_2(self):
+        f = io.StringIO()
+        u = io.StringIO()
+
+        with redirect_stdout(f), redirect_stderr(u):
             resp = request_wrapper(
-                requests.get, ("asdfasdf",), timeout=1, num_retries=2
+                requests.post,
+                args=("https://www.google.com",),
+                kwargs={"json": {"value": 10}},
+                num_retries=1,
+                output_to_stdout=True,
+                timeout=1,
             )
-        except:
-            assert True
+
+        assert "status_code" in str(u.getvalue()) + str(f.getvalue())
+        assert "message" in str(u.getvalue()) + str(f.getvalue())
 
     def test_request_wrapper_pass(self):
         resp = request_wrapper(
             requests.get, ("https://www.google.com",), timeout=1, num_retries=2
         )
+        if resp is None:
```

Comment:
remove this

Reply:
This is in a test - I think this is important to ensure that required behavior does not change. This will save problems in the future if someone forces this.

Reply:
resp will never be none tho, this was added before we removed `raise_errors=True`